feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -16,7 +16,11 @@ Test the abnormal termination of session using notification message:
 
     *   Ensure session state should be `ESTABLISHED`.
     *   Verify BGP capabilities: route refresh, ASN32 and MPBGP.
-
+    *   Enable MD5 authentication on DUT and ATE. Password verification
```

Comment:
The two added bullet points do not fit well to the flow of the test plan. This section is about establishing a session and terminate it abnormally. The `* Establish BGP session` bullet point (line 15) is about establishing the session, and `* Verify BGP session disconnect` (line 24) is about terminating the session abnormally.

The section below is about establishing and terminating a session normally, so the password mismatch test doesn't really fit there either.

The negative password test is about abnormal session establishment, so it should be in its own section. See suggested edit below.

Reply:
Added TestPassword function, Thank you.